earthpy/plot.py

```diff
@@ -14,8 +14,9 @@
 
 
 def colorbar(mapobj, size="3%", pad=0.09):
-    """Adjusts the height of a colorbar to match the axis height. Note that
-    this function will not work properly using matplotlib v 3.0.0 in Jupyter
+    """Adjust colorbar height to match the axis height.
+
```

Comment:
`NOTE: This function requires matplotlib v 3.0.1 or greater or  v 2.9 or lower to run properly. `